docs/UserGuide.md

```diff
@@ -15,31 +15,33 @@ that you might not remember and organise your tasks neatly.
-    - [Deleting a contact : `delete_contact`](#deleting-a-contact-delete-contact)
-    - [Editing a contact : `edit_contact`](#editing-a-contact-edit-contact)
-    - [Locating contacts by name : `find_contact`](#locating-contacts-by-name-find-contact)
-    - [Adding a tag : `add_tag`](#adding-a-tag-add-tag)
-    - [Viewing all tags : `view_tags`](#viewing-all-tags-view-tags)
-    - [Deleting a tag : `delete_tag`](#deleting-a-tag-delete-tag)
-    - [Filtering contacts by tag : `filter_by_tag`](#filtering-contacts-by-tag-filter-by-tag)
-    - [Adding an event : `add_event`](#adding-an-event-add-event)
-    - [Viewing all events : `view_events`](#viewing-all-events-view-events)
-    - [Deleting an event : `delete_event`](#deleting-an-event-delete-event)
-    - [Linking contacts to an event : `link`](#linking-contacts-to-an-event-link)
+    - [Adding a contact : `add_contact`](#adding-a-contact--add_contact)
+    - [Listing all contacts : `view_contacts`](#listing-all-contacts--view_contacts)
+    - [Deleting a contact : `delete_contact`](#deleting-a-contact--delete_contact)
+    - [Editing a contact : `edit_contact`](#editing-a-contact--edit_contact)
+    - [Locating contacts by name : `find_contact`](#locating-contacts-by-name--find_contact)
+    - [Adding a tag : `add_tag`](#adding-a-tag--add_tag)
+    - [Viewing all tags : `view_tags`](#viewing-all-tags--view_tags)
+    - [Deleting a tag : `delete_tag`](#deleting-a-tag--delete_tag)
+    - [Filtering contacts by tag : `filter_by_tag`](#filtering-contacts-by-tag--filter_by_tag)
+    - [Adding an event : `add_event`](#adding-an-event--add_event)
+    - [Viewing all events : `view_events`](#viewing-all-events--view_events)
+    - [Deleting an event : `delete_event`](#deleting-an-event--delete_event)
+    - [Linking contacts to an event : `link`](#linking-contacts-to-an-event--link)
     - [Unlinking contacts from an event: `unlink`](#unlinking-contacts-from-an-event-unlink)
-    - [Selecting an event: `select_event`](#selecting-an-event-select-event)
-    - [Adding a task : `add_task`](#adding-a-task-add-task)
-    - [Viewing help : `help`](#viewing-help-help)
-    - [Deleting a task: `delete_task`](#deleting-a-task-delete-task)
-    - [Marking a task: `mark_task`](#marking-a-task-mark-task)
-    - [Unmarking a task: `unmark_task`](#unmarking-a-task-unmark-task)
-    - [Clearing all entries : `clear`](#clearing-all-entries-clear)
-    - [Returning to home page : `home`](#returning-to-home-page-home)
-    - [Exiting the program : `exit`](#exiting-the-program-exit)
+    - [Selecting an event: `select_event`](#selecting-an-event-select_event)
+    - [Adding a task : `add_task`](#adding-a-task--add_task)
+    - [Deleting a task: `delete_task`](#deleting-a-task-delete_task)
+    - [Marking a task: `mark_task`](#marking-a-task-mark_task)
+    - [Unmarking a task: `unmark_task`](#unmarking-a-task-unmark_task)
+    - [Viewing help : `help`](#viewing-help--help)
```

Comment:
using double -- seems to cause the hyperlinks to fail to function. Perhaps better to keep in its current state

Reply:
oh there was some merge conflicts that happened when I pushed this PR thats why it seemed redundant so perhaps I can redo some of the changes that are actually relevant in another PR